### PR TITLE
Enh add theme switcher

### PR DESCRIPTION
**Pull request by jklymak**
The dark theme works here, but someone who has a machine that can use Calibri and Carlito with Matplotlib will have to generate a version with brighter logo colors.  I'm also not sure how this ends up looking with the main docs.

### CODE REVIEW

**Comment by timhoffm** [comment]
Are we sure, we can/want to reasonably support a dark theme? All plots have a solid white background and will stand out against the dark theme.

**Comment by jklymak** [comment]
Sure I agree we don't necessarily want this.  But thought it was worth playing with quickly.

**Comment by timhoffm** [comment]
I’ve added it to the agenda today.

**Comment by story645** [comment]
So far as I can tell, consensus on the call last week  was it's worth a try, especially if the pydata-sphinx-theme can automatically do the color inversion on the images (which seems to be the case in scipy's dark theme but I'm struggling to find the reference). With the caveat that it's provisional/contingent on the types of bug reports folks file.

**Comment by timhoffm** [comment]
> > automatically do the color inversion on the images
>
> Just inverting images in dark mode seems risky to me?

Yes. That usually looks bad on colored images.

**Comment by story645** [comment]
There are a couple where it might break the semantics, and that can be tracked via bug report, but generally it was good enough. I'm having trouble digging up the example @greglucas found that showed how it works 😕

**Comment by timhoffm** [comment]
Agreed that it looks better than expected. I still have a slight reservation as the plot is our key deliverable not just an illustration, wirting color='blue' and getting yellow is a bit of a stretch. But if people want to inverst time here, I'm not blocking.

**Comment by story645** [comment]
> I think people who care about color will easily realize they need to use the light theme?

Or file a bug report, & if it's only a handful of images it may not be a big deal to generate dark mode versions just for those images.

**Comment by jklymak** [comment]
I don't think there is a bug report to file - I assume scipy is just inverting all the colors in each image.

**Comment by story645** [comment]
> I don't think there is a bug report to file 
 i mean they'll open an issue w/ matplotlib that the labels/colors are wrong in dark mode

**Comment by jklymak** [comment]
Added a couple of plots using a plot directive.  They do not invert.  I haven't bothered setting up a gallery, though we could I suppose.

File: mpl_sphinx_theme/static/css/style.css [hunk 1]
```diff
@@ -9,6 +9,10 @@ html[data-theme="light"] {
 }
 
 html[data-theme="dark"] {
+<<<<<<< HEAD
```
Comment (timhoffm) [inline mpl_sphinx_theme/static/css/style.css:12]: Remove diff markers.

**Comment by timhoffm** [comment]
The dark style needs some further style customization:

- Header colors is to dark
  grafik
- Theme switcher icon is too dark
  grafik
- Logo is not found
  grafik

File: mpl_sphinx_theme/static/css/style.css (hunk 1 above)
Comment (jklymak) [reply mpl_sphinx_theme/static/css/style.css]: Ooops, must have missed a Save...

**Comment by timhoffm** [comment]
Color is fixed. Logo is still missing.

**Comment by jklymak** [comment]
Yes, we needed a dark logo.  I just made this one by hand in Inkscape, and adjusted the brightness until it looked pretty good.

**Comment by jklymak** [comment]
Latest commit makes the header colors match the logos.  This hurts my eyes far less than the unmatched versions we had before, though I think this was controversial?

**Comment by jklymak** [comment]
> though I think this was controversial? 

Maybe less so than I thought - the headers _used_ to match the logo before we switched to pydata-sphinx-theme

**Review by timhoffm — APPROVED** [review]
This is good for a start. Refinement can be done in followup PRs.

**Pull request merged**